app/src/main/res/layout/home_activity.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <android.support.v7.widget.CardView
+                android:id="@+id/card_visitasmedicas"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/height_card_medic"
+                android:layout_gravity="center"
+                android:layout_marginBottom="@dimen/space_cards"
+                android:layout_marginLeft="@dimen/space_margin_cards"
+                android:layout_marginRight="@dimen/space_margin_cards"
+                android:layout_marginTop="@dimen/space_margin_cards"
+
+                android:onClick="visitasMedicas"
+                app:cardCornerRadius="@dimen/corner_radius_card"
+                app:cardElevation="@dimen/card_elevation">
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    android:orientation="vertical">
+
+                    <ImageView
```

Comment:
O card não foi projetado conforme o protótipo. Entretanto, como discutido em sala, o protótipo não segue fielmente as recomendações do Material Design e, sendo assim, o projeto final do card pode ter pequenas variações em relação ao protótipo.

Mesmo assim, na minha opinião, o card fica muito grande da forma como ele está projetado agora. Se não for pra fazer como está no protótipo, eu sugiro remover essa imagem exibida no topo do card, deixando apenas o texto.